Octokit/Models/Request/CollaboratorRequest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class CollaboratorRequest
+    {
+        public CollaboratorRequest()
+        {
+            Permission = Permission.Push;
```

Comment:
I think so.  but from a consistency point of view we dont do optional parameters that much.  I think in this case it's ok to always pass the request object and only have the `ctor` to explicitly state the permission required.  Nothing wrong with being explicit, particularly when its something like granting access to someone